base/docs/basedocs.jl

```diff
@@ -1811,6 +1825,21 @@ Signed
     Unsigned <: Integer
 
 Abstract supertype for all unsigned integers.
+
+All are printed in hexadecimal, with prefix `0x`, 
+and can be entered in the same way.
+
+# Examples
+```
+julia> unsigned(true)
```

Comment:
Feels like an odd example to have as the first one. Imo the examples should go from the most obvious to the more "esoteric"

Reply:
Yes, good point.